### ISSUE TITLE
Where is Tutorials for c++?

### ISSUE THREAD

**Issue by tokagi-rikka** [Microsoft/CNTK#2710]
(I'm sorry for not good at English)

I attempted to try it with c ++, but when I looked at the tutorial, it was Python, and I got murderous intent.
Input X dimension output Y dimension, there are several hidden layers, I would like to build a simple neural network, but would you teach if there is a site explaining it?

**Comment by BenjaminTrapani** [comment]
Here is one example of such a model: https://github.com/Microsoft/CNTK/blob/master/Tests/UnitTests/V2LibraryTests/FeedForwardTests.cpp
There are many other useful examples in the V2LibraryTests directory. Here is another example that creates and trains a CNTK neural network using C++: https://github.com/mne-tools/mne-cpp/blob/master/libraries/deep/deepmodelcreator.cpp

**Comment by tokagi-rikka** [comment]
BenjaminTrapani, Thank you for your reply!

I happily read the source code of the link destination.
As impressions, it turned out that CNTK was too much over the scale of the neural network I was about to build.
Until now I was using Eigen and C ++ AMP and my skill was not able to raise the speed, so I began looking for a library, but with a small amount of commentary on this CNTK the part of the black box is too big I was worried that the cause could not be grasped when something went wrong.

Because the Japanese site dealing with Caffe looked good to understand, I decided to try using Caffe.
http://d.hatena.ne.jp/muupan/20141010/1412895321

As a result I am sorry that I can not meet your effort.

Thank you very much.
